### ISSUE TITLE
test failures with Time-HiRes

### ISSUE THREAD

**Issue by fperrad** [arsv/perl-cross#36]
with `perl-5.25.4` and `perl-5.25.5`

```
../dist/Time-HiRes/t/utime.t ....................................... 1/18 
#   Failed test 'atime set correctly'
#   at t/utime.t line 35.
#          got: '1'
#     expected: '1.111111111'

#   Failed test 'mtime set correctly'
#   at t/utime.t line 36.
#          got: '2'
#     expected: '2.222222222'

#   Failed test 'atime set correctly'
#   at t/utime.t line 44.
#          got: '1'
#     expected: '1.111111111'

#   Failed test 'mtime set correctly'
#   at t/utime.t line 45.
#          got: '2'
#     expected: '2.222222222'

#   Failed test 'File 1 atime set correctly'
#   at t/utime.t line 55.
#          got: '1'
#     expected: '1.111111111'

#   Failed test 'File 1 mtime set correctly'
#   at t/utime.t line 56.
#          got: '2'
#     expected: '2.222222222'

#   Failed test 'File 2 atime set correctly'
#   at t/utime.t line 60.
#          got: '1'
#     expected: '1.111111111'

#   Failed test 'File 2 mtime set correctly'
#   at t/utime.t line 61.
#          got: '2'
#     expected: '2.222222222'

#   Failed test 'File 1 atime set correctly'
#   at t/utime.t line 75.
#     '0.281636953353882'
#         <
#     '0.1'

#   Failed test 'File 1 mtime set correctly'
#   at t/utime.t line 76.
#     '0.281636953353882'
#         <
#     '0.1'

#   Failed test 'File 2 atime set correctly'
#   at t/utime.t line 80.
#     '0.281636953353882'
#         <
#     '0.1'

#   Failed test 'File 2 mtime set correctly'
#   at t/utime.t line 81.
#     '0.281636953353882'
#         <
#     '0.1'
# Looks like you failed 12 tests of 18.
../dist/Time-HiRes/t/utime.t ....................................... Dubious, test returned 12 (wstat 3072, 0xc00)
```

**Comment by fperrad** [comment]
This is an upstream issue, seen on Debian Jessie (armhf) with
```
$ perlbrew install perl-5.25.5
```

**Comment by fperrad** [comment]
This issue is already tracked in <http://rt.cpan.org/Public/Bug/Display.html?id=116127> for `Time-HiRes`.

I caused it because I use an `ext2` filesystem.

**Comment by fperrad** [comment]
Fixed upstream in perl-5.25.6 with Time-HiRes 1.9740
